### PR TITLE
Support for auto answer

### PR DESCRIPTION
**Pull request by areski**
Trying to add support for auto-answer but without success [ref: https://github.com/OpenTelecom/react-sip-phone/commit/5ab38f07800bde4b348820b7a563cc1a066bba21]

I'm just learning TypeScript so I could easily be trap in a noob mistake!

I could use a hand :smiley:

### CODE REVIEW

**Comment by noahmehl** [comment]
We are likely to consider and merge: https://github.com/OpenTelecom/react-sip-phone/pull/38 first so this can be rebased before a full review.

However, I believe that browser compatibility for this may also be an issue:

https://webrtchacks.com/autoplay-restrictions-and-webrtc/

File: example/src/App.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ const App = () => {
           disabledFeatures: disabledFeatures || '', // Will remove feature(s) from application. E.g. settings remoteid
           defaultDial: '',          // (strict-mode only) the default destination. E.g. 1234567890
           sessionsLimit: 3,         // limits amount of sessions user can have active
-          attendedTransferLimit: 2  // limits amount of attendedTransfer sessions user can have active
+          attendedTransferLimit: 2, // limits amount of attendedTransfer sessions user can have active
+          autoAnswer: true         // enable the auto-answer on incoming calls
```
Comment (noahmehl) [inline example/src/App.js]: I definitely appreciate that you added the prop to the example, however my preference would be for this to be false by default.

Comment (areski) [reply example/src/App.js]: I got distracted, also wanted it to false for default, thanks for the help

**Comment by areski** [comment]
For some weird reason, the auto-answer is now working fine, on both Firefox and Chrome...

**Pull request merged**